cylc/flow/option_parsers.py

```diff
@@ -71,23 +71,73 @@
 DOUBLEDASH = '--'
 
 
-ICP_OPTION = {
-    ARGS: ["--initial-cycle-point", "--icp"],
-    KWARGS: {
-        METAVAR: "CYCLE_POINT or OFFSET",
-        HELP:
-            "Set the initial cycle point."
-            " Required if not defined in flow.cylc."
-            "\nMay be either an absolute point or an offset: See"
-            f" {SHORTLINK_TO_ICP_DOCS} (Cylc documentation link).",
-        ACTION: "store",
-        DEST: "icp",
-    }
-}
+class optionSettings():
```

Comment:
Any reason why this breaks naming conventions? Maybe change to `OptionSettings` or `CylcOptionSettings` if we are worried about confusion with option parser module?

Reply:
No.